docs/apidocs/org/simulator/examples/COBRAsolverExample.html

```diff
@@ -125,7 +125,7 @@ <h2 title="Class COBRAsolverExample" class="title">Class COBRAsolverExample</h2>
 <li><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">java.lang.Object</a></li>
```

Comment:
Filename wrong? The files are still called COBRAsolverExample (see also next files below)